apps/paymaster-admin/src/components/Home/index.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+import { StateType } from "@fogo/component-library/useData";
+import { SessionButton } from "@fogo/sessions-sdk-react";
+
+import { useUserData } from "../../client/paymaster";
+
+export const Home = () => {
+  const userData = useUserData();
```

Comment:
If you follow my advice above, generally I'd restructure this component to switch on session state.  If the session state is established, then pass `sessionState` to a child component that then calls `useUserData`.